server/bundler/extensions-loader.js

```diff
@@ -0,0 +1,34 @@
+const existsSync = require( 'fs' ).existsSync;
+const path = require( 'path' );
+const camelCase = require( 'lodash' ).camelCase;
+
+function getReducers( extensions = {} ) {
+	return extensions
+		.filter( ( extension ) => existsSync( path.join( 'client', 'extensions', extension, 'state', 'reducer.js' ) ) )
```

Comment:
Could we use an asynchronous `Promise.all` + `exists` + [`compact`](https://lodash.com/docs/4.17.4#compact) instead of `filter` + `existsSync`?